### DISCUSSION TITLE
VMC 4 UE

### DISCUSSION THREAD

**Discussion by Saff1973** [community/community#122305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear friend,
Any way to make VMC from one plugin or what is the correct way to do it any video? my email is: alisaff28@gmail.com

**Comment by ebndev** [comment]
Hi @Saff1973,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, you might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
